travis/secring.asc

```diff
@@ -0,0 +1,83 @@
+-----BEGIN PGP PRIVATE KEY BLOCK-----
```

Comment:
hmm....... secring.asc is a public or private file? if private, this needs to be a secret, no?